src/plugins/vis_type_vislib/public/vislib/helpers/point_series/_get_point.ts

```diff
@@ -41,7 +41,7 @@ export interface Point {
     table: Table;
     column: number;
     row: number;
-    value: number;
+    value: TableParent['formattedKey'];
```

Comment:
I don't like idea to link types to `TableParent['formattedKey']` here. Can we use RowValue instead of that?

Reply:
Now I used 'string'